administrator/components/com_admin/script.php

```diff
@@ -5048,6 +5048,9 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
 			// Joomla 4.0 Beta 7
 			'/media/vendor/skipto/js/dropMenu.js',
 			'/media/vendor/skipto/css/SkipTo.css',
+			// Joomla 4.0 Beta 8
+			'/administrator/templates/atum/scss/vendor/minicolors/minicolors.scss',
+			'/templates/cassiopeia/scss/vendor/_minicolors.scss'
```

Comment:
```suggestion
			'/templates/cassiopeia/scss/vendor/_minicolors.scss',
```
You have to add back the commas at the end of the last array element. We do that because PHP allows it and it makes maintenance of the array easier, i.e. if people add new stuff to the end, they don't have to add a comma to the last line before that place.